### PR TITLE
Add config dropDownCollapsable

### LINKED ISSUE #21: Collapse/expand customization after selecting a dropdown row
User can opt in to collapse the dropdown or stay expanded after they selected any row

Need to add an option in DropDownConfig with default value to collapse on selecting a row. Also update the DropDownView accordingly

### PR DESCRIPTION
**Pull request by devanshimodha**
# Description

Add dropDownCollapsable in DropDownConfig

Fixes #21 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)\
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe on what device it is tested

* Xcode: Version 12.0.1 
* Model name: iPhone SE (2nd Generation) - iOS 13.5

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

# Reviewers:

Please tag relevant people requesting a review

### CODE REVIEW

File: Example/ViewController.swift [hunk 1]
```diff
@@ -49,6 +49,12 @@ extension ViewController: CustomDropDownDelegate, CustomDropDownDataSource {
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath, data: Any, identifier: Int) {
         
     }
+    
+    func config(identifier: Int) -> DropDownConfig {
```
Comment (tejuamirthi) [inline Example/ViewController.swift:53]: As the default value is true for dropDownCollapsable, we dont need this block of code. Remove this

**Review by tejuamirthi — CHANGES_REQUESTED** [review]

**Comment by tejuamirthi** [comment]
Hey @devanshimodha, as you are working on this issue, can you take a look at #17

**Review by tejuamirthi — APPROVED** [review]

**Comment by tejuamirthi** [comment]
Dropdown collapsable option

**Comment by devanshimodha** [comment]
Hello @tejuamirthi 

I am geetting this error in hacktoberfest portal

"Your PR was submitted to a repository that is not participating in Hacktoberfest. Maintainers of the repository can add the "hacktoberfest" topic to their repository if they wish to participate. Alternatively, an individual PR can be opted-in with a maintainer adding the "hacktoberfest-accepted" label to the PR. "

**Comment by tejuamirthi** [comment]
> Hello @tejuamirthi
> 
> I am geetting this error in hacktoberfest portal
> 
> "Your PR was submitted to a repository that is not participating in Hacktoberfest. Maintainers of the repository can add the "hacktoberfest" topic to their repository if they wish to participate. Alternatively, an individual PR can be opted-in with a maintainer adding the "hacktoberfest-accepted" label to the PR. "

How about now? Is it showing the same?

**Comment by tejuamirthi** [comment]
Hey @devanshimodha 
If you see the same again, please look at the other issues and contribute. Because this pr is merged, I guess it may not have a chance to validate.

**Comment by devanshimodha** [comment]
Done, I'll try to solve other issues too

**Pull request merged**